test/integration/testem/testem.js

```diff
@@ -128,20 +132,27 @@ function setChromeFlags(flags) {
     };
 }
 
-const testemConfig = Object.assign({}, defaultTestemConfig);
-
-if (process.env.RENDER) Object.assign(testemConfig, renderTestemConfig);
-
-if (process.env.CI) {
-    // Set chrome flags for CircleCI to use llvmpipe driver (see https://github.com/mapbox/mapbox-gl-js/pull/10389).
-    const ciTestemConfig = setChromeFlags([ "--disable-backgrounding-occluded-windows", "--ignore-gpu-blocklist", "--use-gl=desktop" ]);
-    Object.assign(testemConfig, ciTestemConfig);
-
-} else if (process.env.RENDER && process.env.USE_ANGLE && ['metal', 'gl', 'vulkan', 'swiftshader', 'gles'].includes(process.env.USE_ANGLE)) {
-    // Allow setting chrome flag `--use-angle` for local development on render/query tests only.
-    // Search accepted values for `--use-angle` here: https://source.chromium.org/search?q=%22--use-angle%3D%22
-    const angleTestemConfig = setChromeFlags([ `--use-angle=${process.env.USE_ANGLE}` ]);
-    Object.assign(testemConfig, angleTestemConfig);
+if (ci) Object.assign(testemConfig, ciTestemConfig);
```

Comment:
Awesome work -  mac passes for me! I am out of town until Tuesday night, but can run on my windows then if you'd like me to check on that platform! So, just checking -  the only custom chrome flags when running on back is  "--disable-backgrounding-occluded-windows" unless specifying USE_ANGLE?

Reply:
Yes, `--disable-backgrounding-occluded-windows` is now the only Chrome flag that's set on all OS's. Setting the other two flags to be Linux specific results in less inconsistencies.

`yarn test` and `yarn test-firefox` should now both work on Windows machines.